src/Notes/InstallWooCommercePayments.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+use Automattic\WooCommerce\Admin\API\Plugins;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Install_WooCommerce_Payments.
+ */
+class Install_WooCommerce_Payments {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Attach hooks.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'install_woocommerce_payments' ) );
+	}
+
+	/**
+	 * Install Woocommerce Payments.
+	 */
+	public function install_woocommerce_payments() {
```

Comment:
Do we need a capability check like [here](https://github.com/woocommerce/woocommerce-admin/blob/1e0061d0df8c177795c58ab9603af95f4aef6610/src/Notes/WooCommercePayments.php#L136), or will it just fail (or… not be shown to begin with?) without it?